scripts/cfos/make_dataset_data_tables.py

```diff
@@ -143,57 +108,98 @@ def load_excel_input(excel, expected_dataset_cols):
         else:
             print(e)
         exit()
-    
+
     print("Success: Excel file has been loaded into a dataframe.")
-    return dataset_metadata_df
+    return raw_dataset_df #dataset_metadata_df
 
 
-def get_expected_columns_list(schema_json, dataset_name):
+def parse_config_file(schema_json):
     """Get list of expected columns in excel based on selected dataset schema."""
```

Comment:
I think this function description needs to be updated (i.e. it no longer returns a list)

Reply:
Done! Good catch.

Reply:
In a similar vein, I just updated how use of the validation only flag is handled. I changed it to this, so that if validate_only is set, it overrides any other parameters. This better reflects the behavior intended, I think. 

`validated_dataset = validate_dataset(dataset=dataset_metadata, schema_dict=schema_dict[args.dataset_name], field_dict=column_dict)
    
    if args.validate:
        print("Validation only run complete")
        quit()

    tsv_filenames_list = generate_load_table_files(validated_dataset)
    `